app/assets/javascripts/user_form.js

```diff
@@ -7,6 +7,29 @@ $(function () {
      $('.form-group.user_company_info').toggle(this.checked);
   }).change(); //ensure visible state matches initially
 
+
+   //update coach availability
+
+   $(function () {
+      var $update_availability = $('[data-behavior="update_availability"]');
+      $update_availability.on('click',function(){
+        var url = "/users/current/update_availability";
```

Comment:
I think hardcoding routes in javascript is somewhat of an issue since it can lead to javascript breaking when those routes changes. This is especially troublesome if these things are not covered by system tests since they then will fail in production. I think the right way to pass the url in this case would probably be to have to url in an additional data field in the element that has the data-behaviour field.